exopy_hqc_legacy/instruments/drivers/visa/agilent_pna.py

```diff
@@ -870,6 +880,12 @@ def trigger_source(self, value):
     @secure_communication()
     def data_format(self):
         """
+        REAL,+32 - Best for transferring large amounts of measurement data.
+                   Can cause rounding errors in frequency data.
+        REAL,+64 - Slower but has more significant digits than REAL,32.
+                   REQUIRED to accurately represent frequency data.
+        ASC,+0 - The easiest to implement, but very slow.
+                 Use when you have small amounts of data to transfer.
         """
         data_format = self.query('FORMAT:DATA?')
         if data_format:
```

Comment:
```suggestion
if "0" in data_format:
    return "ascii"
elif "32" in data_format:
     return "real, 32"
elif "64" in data_format:
    return "real, 64"
```
and keep the else clause

Reply:
Done